README.md

```diff
@@ -1,68 +1,88 @@
+For physical server or virtual machine on-premise or custom cloud deployments:
 
-- [AWS with CloudFormation](/beta/aws-ecsfargate-cfn)
-- ✨ **NEW** [Google Cloud Run](/beta/google-cloud-run)
+### Docker
 
-## Deprecated deployment methods
+- [Docker Swarm using Docker CLI](/docker)
 
-A list of recently-deprecated deployments can be found in [`/deprecated`](./deprecated/). At the time of deprecation, these deployments were fully functional, but will soon become unsupported.
+### Kubernetes
 
-- [**List of deprecated deployment methods**](./deprecated/README.md#deprecated-deployments)
+- [Kubernetes using kubectl CLI](/kubernetes)
+- [Helm Chart using Helm CLI](https://github.com/1Password/op-scim-helm/tree/main/charts/op-scim-bridge#installation)
 
-### Deprecation schedule
+## Choosing a deployment method
 
-When a deployment method is deprecated, we will simultaneously append a deprecation notice to the deployment name listed in this README and move all files associated with the deployment method to [`/deprecated`](./deprecated/).
+The choice of deployment method depends on your available or preferred cloud service provider or infrastructure, as well as the platform and tooling that you use or prefer. All deployment examples are based on the same container image for 1Password SCIM Bridge that is pulled from Docker Hub.
+Here are some additional details to help you make an informed choice:
 
-Deprecated deployments will remain in [`/deprecated`](./deprecated/) for approximately **three months**, after which time they will be deleted. The deletion date of deprecated deployments will be posted in [`/deprecated/README.md`](./deprecated/README.md).
+### Marketplace app deployments
 
-Where possible, we will provide suggested alternatives in [`/deprecated/README.md`](./deprecated/README.md).
+For a guided deployment of 1Password SCIM Bridge that connects to your 1Password account to automatically generate and load credentials from 1Password, choose a marketplace app. These are off-the-shelf deployments that are not customizable, require creating a public DNS record, and have strict networking requirements.
+
+We publish these apps to [DigitalOcean Marketplace](https://marketplace.digitalocean.com/apps/1password-scim-bridge) and [Google Cloud Marketplace](https://console.cloud.google.com/marketplace/product/agilebits-public/op-scim-bridge).
+
+### Containers as a Service deployments
+
+Many cloud service providers offer **Containers as a Service** (or **Platform as a Service**) deployment platforms. These solutions include DNS and TLS certificate management out-of-the-box for a speedier deployment with a much lower expected cost and burden of maintenance. These examples are suitable for use in a production environment, include common configuration options, and can be used as a base for your own custom deployment.
+
+We include examples for Azure Container Apps, Google Cloud Run, and DigitalOcean App Platform.
+
```

Comment:
Moved `Containers as a Service deployments` to be above `Marketplace app deployments`. Adding bullet points and whitespace to increase readability.

```suggestion
### Containers as a Service deployments

Many cloud service providers offer **Containers as a Service** (or **Platform as a Service**) deployment platforms. These solutions offer:
- DNS and TLS certificate management out-of-the-box
- Speedier deployment
- Lower expected cost and burden of maintenance

These examples are suitable for use in a production environment, include common configuration options, and can be used as a base for your own custom deployment.

We include examples for Azure Container Apps, Google Cloud Run, and DigitalOcean App Platform.

### Marketplace app deployments

For a guided deployment of 1Password SCIM Bridge that connects to your 1Password account to automatically generate and load credentials from 1Password, choose a marketplace app. These are off-the-shelf deployments that:
- Are not customizable
- Require creating a public DNS record
- Have strict networking requirements

We publish these apps to [DigitalOcean Marketplace](https://marketplace.digitalocean.com/apps/1password-scim-bridge) and [Google Cloud Marketplace](https://console.cloud.google.com/marketplace/product/agilebits-public/op-scim-bridge).

```

Reply:
Agreed, but:

- needs white space before the list
- changed bullet items to lower case

```suggestion
### Containers as a Service deployments

Many cloud service providers offer **Containers as a Service** (or **Platform as a Service**) deployment platforms. Compared to other platforms, these solutions offer:

- DNS and TLS certificate management out-of-the-box
- speedier deployment
- lower expected cost and burden of maintenance

These examples are suitable for use in a production environment, include common configuration options, and can be used as a base for your own custom deployment.

We include examples for Azure Container Apps, Google Cloud Run, and DigitalOcean App Platform.

### Marketplace app deployments

For a guided deployment of 1Password SCIM Bridge that connects to your 1Password account to automatically generate and load credentials from 1Password, choose a marketplace app. These are off-the-shelf deployments that may be a suitable choice, but:

- are not customizable
- require creating a public DNS record
- have strict networking requirements

We publish these apps to [DigitalOcean Marketplace](https://marketplace.digitalocean.com/apps/1password-scim-bridge) and [Google Cloud Marketplace](https://console.cloud.google.com/marketplace/product/agilebits-public/op-scim-bridge).

```